webapp/channels/src/packages/mattermost-redux/src/selectors/entities/teams.ts

```diff
@@ -372,3 +373,9 @@ export function makeGetBadgeCountForTeamId(): (state: GlobalState, id: string) =
 export function searchTeamsInPolicy(teams: Team[], term: string): Team[] {
     return filterTeamsStartingWithTerm(teams, term);
 }
+
+export const getTeamIdByChannelId = createSelector(
```

Comment:
Could you make this into a plain function? It doesn't need to use `createSelector` because it's just returning a primitive value which doesn't need to be memoized